Pod/Classes/UI/CardTextField+CardInfoTextFieldDelegate.swift

```diff
@@ -39,8 +39,9 @@ extension CardTextField: CardInfoTextFieldDelegate {
         guard let prefillText = prefillText else {
             return
         }
-
-        nextTextField?.text = prefillText
+        
+        let nextTextFieldText = (nextTextField?.text ?? "").stringByAppendingString(prefillText)
+        nextTextField?.delegate?.textField?(nextTextField!, shouldChangeCharactersInRange: NSMakeRange(0, (nextTextField?.text ?? "").characters.count), replacementString: nextTextFieldText)
```

Comment:
Consider this example:

```swift
class A {
    var a: String = "world"
    
    func doSomething(parameter: String) -> String {
        return "hello \(parameter)"
    }
}

var isNotNil: A? = A()
var isNil: A? = nil

print(isNotNil?.doSomething(isNotNil!.a))
print(isNil?.doSomething(isNil!.a))
```

The force unwrap of "isNil" is safe here, since the method is not called at all, if it does not exist. Same above:
`nextTextField?.delegate?.textField?(nextTextField!, ...`
At the point we force unwrap, it is already assured, that nextTextField is not nil, which saves us an extra `if let` :)